src/otx/core/data/transform_libs/mmdet.py

```diff
@@ -0,0 +1,104 @@
+"""Helper to support MMDET data transform functions."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Callable
+
+import numpy as np
+from mmdet.datasets.transforms import (
+    LoadAnnotations as MMDetLoadAnnotations,
+)
+from mmdet.datasets.transforms import (
+    PackDetInputs as MMDetPackDetInputs,
+)
+from mmdet.registry import TRANSFORMS
+from torchvision import tv_tensors
+
+from otx.core.data.entity.base import ImageInfo
+from otx.core.data.entity.detection import DetDataEntity
+
+from .mmcv import MMCVTransformLib
+
+if TYPE_CHECKING:
+    from mmengine.registry import Registry
+
+    from otx.core.config.data import SubsetConfig
+
+
+@TRANSFORMS.register_module(force=True)
+class LoadAnnotations(MMDetLoadAnnotations):
+    """Class to override MMDet LoadAnnotations."""
+
+    def transform(self, results: dict) -> dict:
+        """Transform OTXDataEntity to MMDet annotation data entity format."""
+        if (otx_data_entity := results.get("__otx__")) is None:
+            msg = "__otx__ key should be passed from the previous pipeline (LoadImageFromFile)"
```

Comment:
Is there a specific reason for using `results["__otx__"]`? Can you point out where it is utilized within the code?

Reply:
https://github.com/openvinotoolkit/training_extensions/blob/8c8c50f64571c4b25ddcf5a1276ebaba5a4bcca9/src/otx/core/data/transform_libs/mmcv.py#L36